.nuke/build.schema.json

```diff
@@ -204,6 +204,14 @@
           "type": "boolean",
           "description": "Whether the current build version is a prerelease(for packaging purposes)"
         },
+        "LatestMajorVersion": {
+          "type": "integer",
+          "description": "The current latest tracer version"
+        },
```

Comment:
What is it used for?

Reply:
The schema.json file was automatically updated after running the build script. The parameter was not added by me, I ran the build script to update the schema.

Reply:
Yup, that's my fault, sorry!